### PR TITLE
User can copy JSON config from Playground

### PR DESCRIPTION
**Pull request by beausmith**
After using the playground, it would be nice to have a single click action to copy the JSON config to place into the `.prettierrc` file.

prettier-copy-config

Clicking `Copy JSON` will copy the current options to the clipboard. Here's an example:

```json
{
  "arrowParens": "avoid",
  "bracketSpacing": false,
  "insertPragma": false,
  "jsxBracketSameLine": false,
  "parser": "babylon",
  "printWidth": 80,
  "proseWrap": "preserve",
  "requirePragma": false,
  "semi": false,
  "singleQuote": true,
  "tabWidth": 2,
  "trailingComma": "es5",
  "useTabs": false
}
```

Here's a video showing how it works:

prettier-copy-config

- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by lipis** [comment]
#### https://deploy-preview-5295--prettier.netlify.com/playground/

That's pretty cool. But I think it's better to ignore the default options.

**Comment by beausmith** [comment]
@lipis - As an advanced user, I totally agree that ignoring the defaults is wise. For someone who may be a bit more green — and perhaps thus using the playground to get a sense of exactly how the options work — I felt that having all the options would be helpful. I decided to implement the simplest solution first, and then consider how to implement an option which ignores default options next.

What's would be your ideal solution which ignores default options. Here's what I was considering:
- Copy button ignores defaults (no option for adding defaults)
  - Add optionally a second button "Copy config JSON with defaults".
- Add new section to the "options sidebar", move the "Copy config JSON" button there and add a checkbox with the label "include default options". 
- Add new section to the "options sidebar", display the whole config JSON with a "include default options" checkbox which toggles the defaults in the displayed JSON.

File: website/playground/Playground.js [hunk 1]
```diff
@@ -220,6 +220,9 @@ class Playground extends React.Component {
                     <Button onClick={this.clearContent}>Clear</Button>
                   </div>
                   <div className="bottom-bar-buttons bottom-bar-buttons-right">
+                    <ClipboardButton copy={JSON.stringify(options, null, 2)}>
+                      Copy JSON
+                    </ClipboardButton>
```
[the remark this answers is not in the document]
Comment (beausmith) [reply website/playground/Playground.js]: Sure thing.

**Comment by lipis** [comment]
on a second thought... maybe having all the options there, is not a bad thing.. it's always easier to delete than add if anyone wants to.

**Comment by beausmith** [comment]
@lipis @j-f1 Amended original commit with requested changes.

**Comment by lipis** [comment]
We are always squashing so there was no need..  Thank you for the contribution.

**Comment by j-f1** [comment]
Thanks for contributing!

**Pull request merged**